change/@microsoft-fast-foundation-14d4f603-9786-4d4d-8db4-af050205e43e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

We'll need this to be prerelease in order to keep accurate versioning for our current alphas and betas.

```suggestion
  "type": "prerelease",
```